server/src/main/java/com/amos/server/EventGrabDemo.java

```diff
@@ -87,7 +89,7 @@ public boolean onKey(View v, int k, KeyEvent e) {
                 int action = e.getAction();
                 int keycode = e.getKeyCode();
                 int meta = e.getMetaState();
-                msg = String.format("Keycode: %d Action: %d Meta: %d", keycode, action, meta);
+                msg = String.format(Locale.GERMAN,"Keycode: %d Action: %d Meta: %d", keycode, action, meta);
```

Comment:
wouldn't we rather use English?

Reply:
Same as below, but besides it doesnt makes a difference here.

Reply:
Please use one or the other. It would be better not to hardcode locales.

Keep to using the same locale as in the printed text. In this case english would fit the best.